src/viz/test/tstEnsight_Translator.py

```diff
@@ -54,19 +53,23 @@
 
   # diff binary files if on little endian system
   if tEnsight_Translator.is_little_endian():
+    if any(platform.win32_ver()):
+      diff_prog="fc"
+    else: 
+      diff_prog="diff"
     print("\nSystem is little endian, diffing binary files\n")
     for var in vars:
       # testproblem_binary_ensight directory
       tEnsight_Translator.diff_two_files( "PROJECT_BINARY_DIR", \
         "testproblem_binary_ensight/{0}/data.0001".format(var), \
         "PROJECT_SOURCE_DIR", "bench/{0}.bin.0001".format(var), \
-        diff_name="diff")
+        diff_name="{0}".format(diff_prog))
 
```

Comment:
I think you can just use the diff_prog variable directly here because it's already a string:

diff_name=diff_prog